app/models/article.rb

```diff
@@ -1,29 +1,27 @@
 # frozen_string_literal: true
 
 class Article < ApplicationRecord
+  # this is required for the following function
+  # scope :sorted, -> { order(created_at: :desc) }
+
   VALID_STATUSES = {
     draft: 0,
     published: 1,
     trashed: 2
   }.freeze
 
   LANGUAGES = {
-    ukr: 0,
-    eng: 1
+    ua: 0,
+    en: 1
   }.freeze
 
-  enum status:   VALID_STATUSES, _prefix: true
-  enum language: LANGUAGES, _prefix: true
+  enum status:   VALID_STATUSES
```

Comment:
why you decided to remove prefix? and do you know that also exists suffix?

Reply:
I removed the prefixes because the values of the enums are different. And I decided that there is no point in using them in this case.
Yes, I read the doc about it - https://api.rubyonrails.org/classes/ActiveRecord/Enum.html